src/Confluent.Kafka/Consumer.cs

```diff
@@ -1147,275 +286,123 @@ public void Poll(int millisecondsTimeout)
             }
         }
 
-        /// <summary>
-        ///     Poll for new consumer events, including new messages
-        ///     ready to be consumed (which will trigger the OnMessage
-        ///     event).
-        /// </summary> 
-        /// <remarks>
-        ///     Blocks indefinitely until a new event is ready.
-        /// </remarks>
+        /// <include file='include_docs_consumer.xml' path='API/Member[@name="Poll"]/*' />
         [Obsolete("Use an overload of Poll with a finite timeout.", false)]
         public void Poll()
             => Poll(-1);
 
-
-        /// <include file='include_docs.xml' path='API/Member[@name="Store_Offsets"]/*' />
+        /// <include file='include_docs_consumer.xml' path='API/Member[@name="StoreOffsets"]/*' />
         public List<TopicPartitionOffsetError> StoreOffsets(IEnumerable<TopicPartitionOffset> offsets)
             => kafkaHandle.StoreOffsets(offsets);
 
+        /// <include file='include_docs_consumer.xml' path='API/Member[@name="Commit"]/*' />
+        public CommittedOffsets Commit()
```

Comment:
Adding this because in hindsight I believe a `Commit` method makes more sense than `CommitAsync`, since in most scenarios i think people will just do `CommitAsync().Wait()` and to implement `CommitAsync` we needed to make use of a thread pool thread etc. `Commit` is more straightforward.

you can also argue this is not a big deal, and that there is not a strong need for this method, but I feel it's better to include it than to leave it out.